src/server/routes/search.rs

```diff
@@ -191,7 +191,7 @@ async fn results(
                     let engines: Vec<EngineHandler> = cookie_value
                         .engines
                         .iter()
-                        .filter_map(|name| EngineHandler::new(name))
+                        .filter_map(|name| EngineHandler::new(name).ok())
```

Comment:
Can you please provide the reason behind including this change ? Like I can see, one of the reasons that it helps standardize code which I really appreciate it, but when I tested the app on my local machine I found that it does not return any errors in the logs. So I feel that both the code of the previous and the new code work exactly the same. So I don't see any clear advantage between the two. That's why I would like to know like what was the reason behind having this change ?. 🙂

Reply:
Yes, sorry, it was my bad. I've changed this, so if the engine initialization fails, the error will propagate upwards properly.

Reply:
Yes, thanks :heart: the new changes look much better and as such I am marking this as resolved :slightly_smiling_face: .